src/commands/script.rs

```diff
@@ -272,11 +310,11 @@ fn execute_command(interpreter: Option<&str>, command: &str, toolchain: Option<&
                 .stdout(Stdio::inherit())
                 .stderr(Stdio::inherit())
                 .spawn()
-                .expect(&format!("Failed to execute script using {}", other)),
+                .unwrap_or_else(|_| panic!("Failed to execute script using {}", other)),
```

Comment:
https://rust-lang.github.io/rust-clippy/master/index.html#/expect_fun_call

Reply:
The same rule would apply if eventually [anyhow](https://docs.rs/anyhow/latest/anyhow/) were to be used, but the change is slightly closer, from `context` to `with_context` to me within a closure, that means that the string will only be built if the error happens.